packages/modal/src/components/modal/Modal.stories.tsx

```diff
@@ -215,6 +216,66 @@ export const CustomDraggableModal = () => {
   );
 };
 
+export const ModalWithStickyContentBottom = () => {
+  const [
+    isStickyBottomContentModalOpen,
+    setStickyBottomContentModalOpen,
+  ] = useState(false);
+  return (
+    <div>
+      <PrimaryButton
+        onClick={() => setStickyBottomContentModalOpen(true)}
+        label={"Open sticky bottom content modal"}
+      />
+      <Modal
+        headerText={"Header text"}
+        isOpen={isStickyBottomContentModalOpen}
+        onRequestClose={() => setStickyBottomContentModalOpen(false)}
+        shouldCloseOnOverlayClick
+        width={"750px"}
+        footer={
+          <ModalFooter
```

Comment:
Don't make the user add `ModalFooter`, do that automatically.

Reply:
Yes had not removed it yet its done now.